content/community/news/general/2021-11-10-interchange-update.md

```diff
@@ -0,0 +1,424 @@
+can convert OpenMM objects to file formats understood by other engines, but these steps can have
+rough edges and add complexity to users' workflows. The object of the Interchange project is to
+provide a first-class support for these operations with the OpenFF software stack. This enabled by
+a Python package of the same name that is described below and in its
+[documentation](https://openff-interchange.readthedocs.io/en/latest/index.html#).
+
+This post aims to provide a high-level overview of the aims of the project, its current status,
+present capabilities, and some future directions.
+
+{{< note >}}
+Like much of of OpenFF infrastructure, Interchange is rapidly evolving and details within this blog post are likely to be out of date in the future.
+{{< /note >}}
+
+Code in this post should run if copy-pasted into a Jupyter notebook or Python interpreter. This is
+being written in November 2021 and code snippets intended to be used with Interchange v0.1.3. The API is
+not yet stable; the names, import paths, and structure of key components will likely be updated in
+the future in accordance with user feedback. If you'd like to follow along, see installation
+instructions at the bottom of the page; ``Interchange`` is distributed under the MIT license and all
+functionality is accessible using only free and open-source tools.
+
+
+## Contents of an Interchange object
+
+To emphasize a detail above, the key objective of the Interchange project at present is to make it
+easier to get OpenFF force fields in use in non-OpenMM workflows. The scope of this task is large,
+but it is finite; the
+[SMIRNOFF](https://open-forcefield-toolkit.readthedocs.io/en/0.10.1/users/smirnoff.html)
+specification used by OpenFF force fields provides clarity to many questions of information
+content in force fields, which is key for developing an interface to it. The [OpenFF
+Toolkit](https://open-forcefield-toolkit.readthedocs.io/en/0.10.1/topology.html) also already provides
+convenient representations of molecular topologies and functions for parsing existing data into its
+models. Note that this does not close the door on interfacing with non-OpenFF models, it simply
+provides a starting point. Expanding this work outside of the OpenFF stack is discussed below
+("Future work").
+
+An `Interchange` object stores all the information known about a system; this includes its
+chemistry, how that chemistry is represented by a force field, and how the system is organised in 3D
+space. An `Interchange` object has four components, each of which is also an attribute of the core
+Python class
+
+The `topology` attribute stores chemical information in a graph-like structure, without specific information
+about forces from the force field. Currently, this attribute is literally a
+[`Topology`](https://open-forcefield-toolkit.readthedocs.io/en/latest/api/generated/openff.toolkit.topology.Topology.html#openff.toolkit.topology.Topology)
```

Comment:
Other RTD links go to `0.10.1`, but this one goes to `latest` - I don't have a strong opinion on which to point to, but it'd be good to standardize throughout the blog post.

Reply:
I've chosen to explicitly link to the current versions. I can easily be convinced otherwise if Josh disagrees while reviewing